components/frontend-react/src/App.js

```diff
@@ -1,31 +1,37 @@
 import React from 'react';
 import { Switch, Route, Redirect } from 'react-router-dom';
+import { connect } from 'react-redux';
+
 import LoginComponent from './containers/LoginComponent';
 import AdminComponent from './containers/AdminComponent';
 import Dashboard from './containers/Dashboard';
 import QueuedComponent from './containers/QueuedComponent';
 import CompletedComponent from './containers/CompletedComponent';
 
-const App = () => (
+const App = (props) => (
```

Comment:
let's change this to 
```
const App = ({isloggedIn}) => (
```
Then you can use, 

```
<Route exact path='/' isloggedIn={isloggedIn} component={LoginComponent} />
```

Reply:
Yeah every time at the beginning of the function or render distrust the state and props before use.